From a7479ae20b3725616e345e8c6423f5d56d89624b Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 13 Dec 2023 10:26:19 +0000
Subject: [PATCH] ARM: dts: bcm2711-rpi-400: Don't delete the ACT LED

The Pi 400 only has one user-accessible LED (green), which is intended
to be used as a power indication. Therefore there is no activity LED.
However, a user may wish to remap the activity LED functionality to a
GPIO in the 40-way header, so preserve the led-act node in a disabled
state.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/bcm2711-rpi-400.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/boot/dts/broadcom/bcm2711-rpi-400.dts b/arch/arm/boot/dts/broadcom/bcm2711-rpi-400.dts
index de36357cd73f..45c3676b3d16 100644
--- a/arch/arm/boot/dts/broadcom/bcm2711-rpi-400.dts
+++ b/arch/arm/boot/dts/broadcom/bcm2711-rpi-400.dts
@@ -36,8 +36,6 @@ &led_pwr {
 	gpios = <&gpio 42 GPIO_ACTIVE_HIGH>;
 };
 
-/delete-node/ &led_act;
-
 &pm {
 	/delete-property/ system-power-controller;
 };
-- 
2.34.1

